workflow/scripts/_backend_atlite.py

```diff
@@ -0,0 +1,85 @@
+"""Backend for capacity factor calculations using atlite."""
+
+from pathlib import Path
+
+import atlite
+import geopandas as gpd
+import rasterio as rio
+import xarray as xr
+
+
+def cf_agg_from_point_layout(
```

Comment:
slightly wordier is ok if it adds clarity?

```suggestion
def cf_aggregated_from_point_layout(
```